temporalcli/commands.taskqueue.go

```diff
@@ -29,9 +29,20 @@ type pollerRowType struct {
 	RatePerSecond  float64   `json:"ratePerSecond"`
 }
 
+type statsRowType struct {
+	BuildID                 string        `json:"buildId"`
+	TaskQueueType           string        `json:"taskQueueType"`
+	ApproximateBacklogCount int64         `json:"approximateBacklogCount"`
+	ApproximateBacklogAge   time.Duration `json:"approximateBacklogAge"`
```

Comment:
What's the type of age in json? is it a number? if so, we need to qualify the name with something as "approximateBacklogAgeSeconds".

Reply:
We are returning a Duration (effectively, nanoseconds) which gets printed for JSON. Yes, it is a number. Sample output - 

``` "approximateBacklogAge": 4108891000, ```

Reply:
As discussed offline, the type for JSON was changed from `time.Duration` to `string`. Thus, I think it's okay to keep the same name.